dev/com.ibm.ws.microprofile.reactive.messaging_fat/fat/src/com/ibm/ws/microprofile/reactive/messaging/fat/kafka/framework/ExtendedKafkaReader.java

```diff
@@ -50,6 +50,10 @@ public ExtendedKafkaReader(KafkaConsumer<K, V> kafkaConsumer, String topic) {
      * @return the list of records received
      */
     public List<ConsumerRecord<K, V>> waitForRecords(int count, Duration timeout) {
+        return waitForRecords(count, timeout, true);
+    }
+
+    public List<ConsumerRecord<K, V>> waitForRecords(int count, Duration timeout, boolean assertNumberOfRecords) {
```

Comment:
Can we have two separate methods instead of a boolean argument.

Maybe
```
readRecords(int maxCount, Duration timeout);
```
and 
```
assertReadRecords(int count, Duration timeout)
```

Reply:
Done